db/migrate/20170816144924_change_division_on_questions.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeDivisionOnQuestions < ActiveRecord::Migration
+  def up
+    execute("UPDATE loan_questions SET division_id=99")
```

Comment:
@linesarefuzzy Is 99 the correct id across environments or should a variable be used here?